ultra/ultra/env/ultra_env.py

```diff
@@ -55,23 +56,24 @@ def __init__(
         eval_mode=False,
         ordered_scenarios=False,
     ):
-        self.timestep_sec = timestep_sec
-        self.headless = headless
+        self.agent_specs = agent_specs
         self.scenario_info = scenario_info
-        self.scenarios = self.get_task(scenario_info[0], scenario_info[1])
-        if not eval_mode:
-            _scenarios = glob.glob(f"{self.scenarios['train']}")
-        else:
-            _scenarios = glob.glob(f"{self.scenarios['test']}")
+        self.headless = headless
+        self.timestep_sec = timestep_sec
+        self.seed = seed
+        self.eval_mode = eval_mode
+        self.ordered_scenarios = ordered_scenarios
+
+        self.scenarios = self.get_scenarios(scenario_info)
```

Comment:
Do these have to be instance members? I don't think it matters too much if the `ordered_scenarios` is a member of the instance (since it is only a boolean type), but I can imagine the `scenarios` variable can be quite large; and if we only use it in the constructor, it may just be wasted space. Let me know your thoughts.

Reply:
Yes, I was thinking that the headless, timestep_sec and seed parameters are defined to env params, thus I thought it would right to include them as instance members. However for the ordered_scenarios and the agent_specs it would be okay to keep them as regular variables

Reply:
I believe `self.scenarios` can also be changed to just a local variable?